CONTRIBUTING.md

```diff
@@ -0,0 +1,177 @@
+# Contributing to libseccomp-rs
+
+We're really glad you're interested in contributing to libseccomp-rs!
+This document has a few pointers and guidelines to help you get started.
+It is not perfect, and there will always be exceptions to the rules described
+here, but by following the instructions below you should have a much easier time
+getting your work merged with the upstream project.
+
+To have a welcoming and inclusive project, libseccomp-rs uses the Rust project's
+[Code of Conduct][code-of-conduct]. All contributors are expected to follow it.
+
+[code-of-conduct]: https://www.rust-lang.org/conduct.html
+
+## Issues
+
+We use [GitHub's issue tracker][github-issue].
+
+[github-issue]: https://github.com/libseccomp-rs/libseccomp-rs/issues
+
+### Bug Reports
+
+Before submitting a new bug report, please search existing issues to see if
+there's something related. If not, just [open a new issue][new-issue]!
+
+The more information you can give in your issue, the easier it is to figure out
+how to fix it. For libseccomp-rs, this will likely include the
+[upstream libseccomp][upstream-libseccomp] version, the Linux kernel version ,and
+the architecture. In addition, it is incredibly helpful if you can describe/include
+a reproducer for the problem in the description as well as instructions on how to
+test for the bug.
+
+[new-issue]: https://github.com/libseccomp-rs/libseccomp-rs/issues/new
+[upstream-libseccomp]: https://github.com/seccomp/libseccomp
+
+### Feature / API Requests
+
+If you'd like a new API or feature added, please open a new issue requesting it.
+As with reporting a bug, the more information you can provide, the better.
+
+## Pull Requests
```

Comment:
When we have a MSRV (#135) we should add a note here.

Reply:
Good.